SYCL/ESIMD/lsc/lsc_usm_store_u8_u16.cpp

```diff
@@ -25,6 +25,11 @@ template <int TestCastNum, typename T> bool tests() {
   passed &= test<TestCastNum + 4, T, 4, 4, 1, 256, true,
                  lsc_data_size::default_size, cache_hint::none,
                  cache_hint::none, __ESIMD_NS::overaligned_tag<8>>();
+  if constexpr (sizeof(T) == 1) {
```

Comment:
Thank you for adding this 'if constexpr (sizeof(T) == 1)' here, but shouldn't similar checks be added for the calls at the line 22 (sizeof(T) <= 4) and on the line 25 (sizeof(T) <= 2) ? 
How does that test pass when T=int32_t is passed and attempt to store 256 int32_t elements is done?

Reply:
Right, I missed the fact that this test is for bytes/shorts only. Thank you.